packages/datagateway-download/src/downloadApi.ts

```diff
@@ -10,14 +10,15 @@ import {
   handleICATError,
 } from 'datagateway-common';
 
-// TODO: get URLs from settings or something...
-const topcatUrl = 'https://scigateway-preprod.esc.rl.ac.uk:8181/topcat';
-const apiUrl = 'http://scigateway-preprod.esc.rl.ac.uk:5000';
-const idsUrl = 'https://scigateway-preprod.esc.rl.ac.uk:8181/ids';
-
-export const fetchDownloadCartItems: () => Promise<DownloadCartItem[]> = () => {
+export const fetchDownloadCartItems: (
+  facilityName: string,
+  downloadApiUrl: string
```

Comment:
I know I'm making yet another comment about this, but now the call signatures are different! I think we should have all these functions accept a `settings` object parameter so that all the functions are called in similar ways, rather than only specific ones using the settings object way and others using the named parameter way.

Reply:
@louise-davies at the time I did not want to pass those parameters within an object due to the fact that the only parameters the function would take in is settings. However, they can all be passed in as objects to add uniformity to the code.

Reply:
Yeah, it makes sense for the ones that only take settings, the problem is the mismatch now. Plus it's no real different really.

Reply:
@louise-davies I have resolved this in https://github.com/ral-facilities/datagateway/pull/114/commits/d961e3c81840b98978b853bc08ee4d04fbd22339.